lib/versions.xml

```diff
@@ -1898,6 +1898,18 @@
 				</licenses>
 			</library>
 			<library>
+				<file-name>lib/portal/guava.jar</file-name>
+				<version>18.0</version>
+				<project-name>Guava</project-name>
+				<project-url>http://code.google.com/p/guava-libraries</project-url>
```

Comment:
The [actual database] (https://publicsuffix.org/list/effective_tld_names.dat) of public suffix is a bit difficult to use (it would require us to parse the file), so we decided to use Google's Guava library. There's two other libraries out there, but both projects are only maintained by a single author.